meteor-app/imports/ui/member-main-container.js~HEAD

```diff
@@ -1,46 +0,0 @@
-import { Meteor } from 'meteor/meteor'
```

Comment:
I'm presuming that this file being in the repo is my mistake, as I must have let it through in a PR. I suppose it is some kind of merge conflict file? So probably a good catch to delete it